src/iOS/AppDelegate.cs

```diff
@@ -220,7 +220,7 @@ public override void OnActivated(UIApplication uiApplication)
 
         public override void WillEnterForeground(UIApplication uiApplication)
         {
-            _messagingService.Send("resumed");
+            _messagingService?.Send("resumed");
```

Comment:
This will run when the app is transitioning from Background to Active state, if this is null, the app memory was garbage collected so we need to re-init everything otherwise, even if this doesn't crash, the rest of the app might be unusable. 

I suggest merging it for now but revisiting for the IoC / ServiceLocator refactor.

Reply:
yes, totally agree